.github/workflows/actions.yml

```diff
@@ -190,7 +190,8 @@ jobs:
           sw_vers
       - name: Get Dependencies
         run: |
-          brew install cmake gettext libdeflate libevent libpsl miniupnpc ninja node pkg-config
+          brew uninstall -f pkg-config pkg-config@0.29.2
+          brew install cmake gettext libdeflate libevent libpsl miniupnpc ninja node pkgconf
```

Comment:
Maybe add `--formulae` to `brew install` as brew suggests as you are already touching this?
We do not deal with casks here, so I don't see downsides to being more explicit here.